src/pixee-platform.ts

```diff
@@ -5,14 +5,23 @@ import FormData from "form-data";
 import { TOOL_PATH } from "./inputs";
 import {getGitHubContext, getRepositoryInfo} from "./github";
 
-export async function uploadInputFile(tool: TOOL_PATH, file: string) {
-  const fileContent = fs.readFileSync(file, "utf-8");
+export async function uploadInputFile(tool: TOOL_PATH, files: Array<string>) {
   const form = new FormData();
-  form.append("file", fileContent);
-  const pixeeUrl = core.getInput("pixee-api-url");
 
+  const path = require('path');
+
+  // Append each file to the form data
+  files.forEach(file => {
+    const fileContent = fs.readFileSync(file);
+  const baseFilename = path.basename(file);
```

Comment:
Need to fix indentation here I think, although you probably don't need the intermediate variable at all.